test/lib/s3_file_handler_test.rb

```diff
@@ -10,7 +10,7 @@ class S3FileUploaderTest < ActiveSupport::TestCase
   end
 
   test "has the expected filename and content" do
-    s3_file = S3FileUploader.save_file_to_s3("test_file_name.txt", "hello, world\n")
+    s3_file = S3FileHandler.save_file_to_s3("test_file_name.txt", "hello, world\n")
```

Comment:
This consolidation of code is a nice idea 👍  shame there aren't any tests to do with getting the file from s3 - could you write some?